acceptance/suites/tests/00_smoke/validate-syslog-logback.rb

```diff
@@ -1,9 +1,9 @@
 test_name 'SERVER-1215: Validate that logback can be configured to work with syslog'
 
 if options[:type] == 'pe' then
-  servicename   = 'pe-puppetserver'
+  service_name   = 'pe-puppetserver'
```

Comment:
Can we handle the service name selection by just using `options['puppetservice']` like we do here?

https://github.com/puppetlabs/puppet-server/blob/puppet-server-2.3.1/acceptance/suites/tests/010-puppetserver-cli/subcommand/foreground.rb#L8